inc/Utf8/Sort.php

```diff
@@ -52,6 +55,19 @@ protected static function getCollator($reload = false)
         return self::$collator[$lc];
     }
 
+    /**
+     * Enable or disable the use of the intl extension collator
+     *
+     * This is mostly used for testing and should not be used in normal code
+     *
+     * @param bool $use
+     */
+    public static function useIntl($use = true)
+    {
+        self::$useIntl = $use;
+        self::$collator = [];
```

Comment:
There is no need to empty the collator array. The very same collators would be created later, after `useIntl(true)`. After `useIntl(false)` the contents of the array are irrelevant.